src/SimulationControl/SpheralController.py

```diff
@@ -552,7 +555,10 @@ def dropRestartFile(self):
         import time
         start = time.clock()
         fileName = self.restartBaseName + "_cycle%i" % self.totalSteps
-        file = self.restartFileConstructor(fileName, Create)
+        if self.restartFileConstructor is SidreFileIO:
+            file = self.restartFileConstructor(fileName, Create, self.restartFileCount)
+        else:
+            file = self.restartFileConstructor(fileName, Create)
```

Comment:
As above, default to `(filename, access)` here even with Sidre. Use `(filename, access, count)` only when `restartFileCount` is not `None` and we are using `Sidre`